shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -1377,6 +1385,20 @@ void updateSemantics(@NonNull ByteBuffer buffer, @NonNull String[] strings) {
                 event.getText().add(object.label);
                 sendAccessibilityEvent(event);
             }
+
+            // If the object is the input-focused node, then tell the reader about it, but only if
+            // it has changed since the last update.
+            if (inputFocusedSemanticsNode != null && inputFocusedSemanticsNode.id == object.id &&
+                (lastInputFocusedSemanticsNode == null || lastInputFocusedSemanticsNode.id != inputFocusedSemanticsNode.id)) {
+                lastInputFocusedSemanticsNode = inputFocusedSemanticsNode;
```

Comment:
nit: the indentation here is strange making it hard to see where the condition ends and where the body of the if starts.

Reply:
Fixed.